tests/Backend/NetworkPolicies/SnowflakeNetworkPoliciesTest.php

```diff
@@ -0,0 +1,134 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\Test\Backend\NetworkPolicies;
+
+use Keboola\Db\Import\Exception;
+use Keboola\StorageApi\Workspaces;
+use Keboola\Test\Backend\Workspaces\Backend\WorkspaceBackendFactory;
+
+class SnowflakeNetworkPoliciesTest extends NetworkPoliciesTestCase
+{
+    public function testAccessWithoutNetworkPolicy(): void
+    {
+        $verifiedToken = $this->_client->verifyToken();
+
+        $organizationNetworkPolicy = $this->organizationNetworkPolicyName($verifiedToken['organization']['id']);
+        $this->assertNetworkPolicyNotExists($organizationNetworkPolicy);
+
+        $workspaces = new Workspaces($this->_client);
+        $workspace = $workspaces->createWorkspace(['backend' => 'snowflake'], true);
+
+        $this->assertDontHaveNetworkPolicyEnabled($workspace['connection']['user'], $organizationNetworkPolicy);
```

Comment:
muze byt, ale jak jsem psal v connection , o tech user/org network policies jeste moc nevime

Reply:
dohodli jsme se na slacku, ze to tak zatim nechaem https://keboolaglobal.slack.com/archives/C0553EM2RGX/p1727347743541479?thread_ts=1727345995.859289&cid=C0553EM2RGX